1001-2000/LIT1799Lifeof.xml

```diff
@@ -34,17 +33,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
+        <abstract>
+            <p>A collection of homilies for each month of the year
+               describing the life and death of John the Baptist.
+                 It seems to be a rare composition, and it
+                  is different from a work of a similar structure
+                 contained in <ref type="mss" corresp="ESehpp007"/>, and another one
+                 in the edition
+                 <bibl><ptr target="bm:bm:Tarraqan2003GadlaYohannes"/></bibl>.</p>
+        </abstract>
          <langUsage><language ident="en">English</language></langUsage>
```

Comment:
add gez

Reply:
ok

Reply:
not added?
should be 
`<langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>`

Reply:
I did!